kitsune/questions/forms.py

```diff
@@ -315,36 +299,13 @@ def __init__(self, product=None, *args, **kwargs):
                 (key, value["name"]) for key, value in product["categories"].items()
             ]
             category_choices.insert(0, ("", "Please select"))
-            category_field = forms.ChoiceField(
-                label=CATEGORY_LABEL,
-                choices=category_choices,
-                error_messages={
-                    "required": MSG_CATEGORY_REQUIRED
-                }
-            )
-            self.fields["category"] = category_field
-
-        # Collect user agent only when making a question for the first time.
-        # Otherwise, we could grab moderators' user agents.
-        self.fields["useragent"] = forms.CharField(
-            widget=forms.HiddenInput(), required=False
-        )
-
-        self.fields["notifications"] = forms.BooleanField(
-            label=NOTIFICATIONS_LABEL,
-            initial=True,
-            required=False
-        )
+            self.fields["category"].choices = category_choices
 
-    def save(self, user, locale, product, product_config):
-
-        question = Question(
-            creator=user,
-            title=self.cleaned_data["title"],
-            content=self.cleaned_data["content"],
-            locale=locale,
-            product=product
-        )
+    def save(self, user, locale, product, product_config, *args, **kwargs):
+        question = super(NewQuestionForm, self).save(commit=False, *args, **kwargs)
```

Comment:
Why are you using `commit=False` here?

Reply:
Otherwise it errors because the creator_id column is blank

Reply:
Wouldn't something like this work:
```python
self.instance.creator = user
super(NewQuestionForm, self).save()
```

Reply:
That's clever, update incoming